soul-client/soul-client-dubbo/soul-client-alibaba-dubbo/src/main/java/org/dromara/soul/client/alibaba/dubbo/AlibabaDubboServiceBeanPostProcessor.java

```diff
@@ -108,6 +115,8 @@ private MetaDataRegisterDTO buildMetaDataDTO(final ServiceBean<?> serviceBean, f
         String path = contextPath + soulDubboClient.path();
         String desc = soulDubboClient.desc();
         String serviceName = serviceBean.getInterface();
+        String host = org.apache.commons.lang3.StringUtils.isBlank(this.host) ? IpUtils.getHost() : this.host;
```

Comment:
modify this utils

Reply:
becase nacos temporary node is service instance, it must contain host and port.

nacos function like this:

```java
void registerInstance(String serviceName, String ip, int port) throws NacosException;

void registerInstance(String serviceName, String ip, int port, String clusterName) throws NacosException;

void registerInstance(String serviceName, Instance instance) throws NacosException;
```

Other nacos struct is:

soul.register.service.{rpcType} 

contextpath as a instance in "soul.register.service.{rpcType}",  so I can subscribe it. Add host and port as a unique identifier

Reply:
spring cloud client ，may be modify

2. stringUtils maybe replice

Reply:
o, like this:

```java
        String host = StringUtils.isBlank(this.host) ? IpUtils.getHost() : this.host;
        int port = StringUtils.isBlank(this.port) ? -1 : Integer.parseInt(this.port);
```

Reply:
String host = org.apache.commons.lang3.StringUtils.isBlank(this.host)  ，why  not import

Reply:
Spring cloud bean post use: org.apache.commons.lang3.StringUtils, so don't need import.

orther like grpc, user org.springframework.util.StringUtils and org.apache.commons.lang3.StringUtils

so occur some code like: org.apache.commons.lang3.StringUtils.isBlank(this.host)